flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/GenericInMemoryCatalog.java

```diff
@@ -720,6 +720,25 @@ public CatalogTableStatistics getPartitionStatistics(
         return result != null ? result.copy() : CatalogTableStatistics.UNKNOWN;
     }
 
+    /**
+     * For in memory catalog, just iteratively call {@link #getPartitionStatistics(ObjectPath,
+     * CatalogPartitionSpec)} for each partition.
+     */
+    @Override
+    public List<CatalogTableStatistics> bulkGetPartitionStatistics(
```

Comment:
the implementation is totally same the the default one, I think we can remove it